web/src/settings_emoji.js

```diff
@@ -126,6 +126,13 @@ export function populate_emoji() {
             }
             return "";
         },
+        callback_after_render() {
+            // While rendering a list, the ListWidget at the end of the provided list
+            // checks if it needs to render a message in case the container is empty.
+            // However, the provided emoji list might contain deactivated emojis that will
+            // not be displayed. Therefore, an additional check is required here.
+            ListWidget.render_empty_list_message_if_needed($emoji_table);
```

Comment:
@sahil839, Fixed the issue with the 'Custom emoji' panel. Please take a look. Thanks!